frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.tsx

```diff
@@ -621,8 +622,13 @@ export const TableInteractiveInner = forwardRef(function TableInteractiveInner(
     const isNotebookPreview = queryBuilderMode === "notebook";
     const isModelEditor = queryBuilderMode === "dataset";
     const hasObjectDetail =
-      !(isPivoted || hasAggregation || isNotebookPreview || isModelEditor) &&
-      !isEmbeddingSdk;
+      !(
```

Comment:
I don't love how I did this. Basically we needed an in between of "isDashboard" and "isNotebookPreview" if I recall correctly. Would be nice if we just had functionality as props, rather than some switches that do arbitrary things